docs/docs-content/clusters/public-cloud/aws/eks.md

```diff
@@ -1,172 +1,270 @@
+4. Ensure the **Cluster Status** field displays **Running**.
 
+<br />
 
-## EKS Cluster Secrets Encryption
+## Enable Secrets Encryption for EKS Cluster
 
 Palette encourages using AWS Key Management Service (KMS) to provide envelope encryption of Kubernetes secrets stored in Amazon Elastic Kubernetes Service (EKS) clusters. This encryption is 
-a defense-in-depth security strategy to protect sensitive data such as passwords, docker registry credentials, and TLS keys stored as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/). 
+a defense-in-depth security strategy to protect sensitive data such as passwords, docker registry credentials, and Transport Layer Security (TLS) keys stored as [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/). 
+
+You can enable secrets encryption when you create an EKS cluster by toggling the **Enable encryption** button and providing the Amazon Resource Name (ARN) of the KMS key. The **Enable encryption** option is available on the cluster creation wizard's **Cluster Config** page for EKS.
 
 ### Prerequisites
 
-* KMS key created in the AWS account.
-* KMS key is of the type symmetric.
-* KMS key policy permits the following actions; encrypt and decrypt.
+- An AWS account added to Palette. Review [Add AWS Account](add-aws-accounts.md) for guidance.
+
+- IAM user or role has attached policies listed in [Required IAM Policies](required-iam-policies.md).
+
+- A **PaletteControllersEKSPolicy** created in AWS and attached to the IAM user or role that Palette is using. To create this policy, refer to [Controllers EKS Policy](required-iam-policies.md#controllers-eks-policy).
+
+
+### Configure KMS Key
+
+Use the following steps to configure a KMS key.
+
+1. In AWS, locate the Key Management Service. 
+
+2. Select the region where your Palette EKS cluster is deployed.
+
+:::caution
+
+Ensure you create the KMS key in the same region as your Palette EKS cluster. Alternatively, you can create a multi-region KMS key that can be used across different regions. To learn how to create a multi-region key, review Amazon’s [Multi-Region Keys in AWS KMS](https://docs.aws.amazon.com/kms/latest/developerguide/multi-region-keys-overview.html) reference guide.
+
+::: 
+
+3. Create a key of type **Symmetric** and with usage **Encrypt and decrypt**.
+
+4. Ensure the IAM user or role that Palette is using has a policy attached with the following required IAM permissions. 
+
+  ```json hideClipboard
+  kms:CreateGrant,
+  kms:ListAliases,
+  kms:ListKeys,
+  kms:DescribeKeys
+  ```
+   
+   Example:
+
+   ```json
+   {
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "Statement1",
+            "Effect": "Allow",
+            "Action": [
+                "kms:ListKeys",
+                "kms:ListAliases",
+                "kms:DescribeKey",
+                "kms:CreateGrant"
+            ],
+            "Resource": "*"
+        }
+    ]
+  }

```

Comment:
```suggestion
  {
      "Sid": "Allow Palette use of the KMS key",
      "Effect": "Allow",
      "Principal": {
          "AWS": "arn:aws:iam::123456789:user/REPLACE_ME"
      },
      "Action": [
          "kms:Encrypt",
          "kms:Decrypt",
          "kms:ReEncrypt*",
          "kms:GenerateDataKey*",
          "kms:DescribeKey"
      ],
      "Resource": "*"
  },
```
This what the key policy should look like. The user just needs replace the account ID `123456789` and the user name/role name at the end.